Sources/Locker/Helpers/LockerHelpers.swift

```diff
@@ -155,17 +155,18 @@ extension LockerHelpers {
     }
 
     static var keyKeychainServiceName: String {
-        return "\(LockerHelpers.bundleIdentifier)_KeychainService"
+        guard let service = UserDefaults.standard.object(forKey: keyCustomKeychainService) as? String
+        else { return "\(LockerHelpers.bundleIdentifier)_KeychainService" }
+        
+        return service
     }
 
     // MARK: - Device list
 
     static func fetchNewDeviceList() {
-    #if !targetEnvironment(simulator)
```

Comment:
I would consider keeping this check in here. The main reason is that we want to keep fetching the data from the apiary as low as possible. 
Apiary has a limit for fetching the data, and as we don't need that info for the simulator, maybe we can skip fetching them for the unsupported devices.